_data/other.yml

```diff
@@ -114,8 +114,11 @@ websites:
       url: https://mandrill.com/
       img: mandrill.png
       tfa: Yes
+      sms: Yes
       software: Yes
       hardware: Yes
+      exceptions:
+          text: "Phone number must be in the United States, the United Kingdom, Canada or some Caribbean islands."
```

Comment:
Or "SMS only available in the United States, the United Kingdom, Canada or some Caribbean islands."